srcpkgs/fatresize/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'fatresize'
+pkgname=fatresize
+version=1.1.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="libparted-devel"
+depends="parted"
+short_desc="Utility to resize FAT filesystems using libparted"
+maintainer="Anjandev Momi <anjan@momi.ca>"
+license="GPL-3.0-or-later"
+homepage="https://sourceforge.net/projects/fatresize"
```

Comment:
Just make homepage [https://github.com/ya-mouse/fatresize](https://github.com/ya-mouse/fatresize). The Sourceforge project page offers no significant information about the project, while the Github page at least shows the README.